### ISSUE TITLE
modules/python-modules/test_pymodules.sh fails on FreeBSD

### ISSUE THREAD

**Issue by czanik** [syslog-ng/syslog-ng#4250]
modules/python-modules/test_pymodules.sh fails on FreeBSD. This was the first time I used --with-python-packages=none in the FreeBSD ports Makefile

```
PASS: tests/loggen/tests/test_loggen_filereader
FAIL: modules/python-modules/test_pymodules.sh
============================================================================
Testsuite summary for syslog-ng 3.38.1.383.g0bb540b
============================================================================
# TOTAL: 187
# PASS:  186
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
============================================================================
gmake[4]: *** [Makefile:25284: test-suite.log] Error 1
gmake[3]: *** [Makefile:25392: check-TESTS] Error 2
gmake[2]: *** [Makefile:26931: check-am] Error 2
gmake[1]: *** [Makefile:25169: check-recursive] Error 1
gmake: *** [Makefile:26933: check] Error 2
root@fb131:~/syslog-ng/work/syslog-ng-3.38.1.383.g0bb540b # modules/python-modules/test_pymodules.sh 
modules/python-modules/test_pymodules.sh: MAKEFLAGS=: not found
root@fb131:~/syslog-ng/work/syslog-ng-3.38.1.383.g0bb540b # 
```

**Comment by bazsi** [comment]
Is this still open?

**Comment by czanik** [comment]
Testing again next week. ```make check``` is now completely broken on FreeBSD, but it could also be a ports problem, as we have seen earlier. From the logs it seems that the ```criterion``` package does not install all dependencies, and even after installing missing dependencies by hand, 175 of 180 tests coredump :/

**Comment by bazsi** [comment]
There's a criterion issue I just fixed that was caused by a nanopb 0.4.7
update.

**Comment by czanik** [comment]
OK. A ```criterion``` update landed in FreeBSD ports a few days ago, and now everything is green:
```
============================================================================
Testsuite summary for syslog-ng 4.1.1.72.gb0565cb
============================================================================
# TOTAL: 191
# PASS:  191
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
```
Closing it for now.
